From 0a3a6eb97a765438ae34fbec46fe62c9bcdf7a78 Mon Sep 17 00:00:00 2001
From: XenServer <xenserver@cloud.com>
Date: Wed, 5 Mar 2025 22:31:00 +0000
Subject: [PATCH 4/5] Sync with xenserver-release-v8.4.0-17.tar.gz

* Thu Mar 13 2025 Gerald Elder-Vass <gerald.elder-vass@cloud.com> - 8.4.0-18
- CA-407370: Separate rsyslog configs for xenserver and customer

(yes, the tarball version doesn't match the changelog entry, but both
the tarball version and the SRPM release number are right)

Signed-off-by: Samuel Verschelde <stormi-xcp@ylix.fr>
---
 src/common/etc/rsyslog.d/xenserver.conf | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/common/etc/rsyslog.d/xenserver.conf b/src/common/etc/rsyslog.d/xenserver.conf
index be87229..d59c633 100644
--- a/src/common/etc/rsyslog.d/xenserver.conf
+++ b/src/common/etc/rsyslog.d/xenserver.conf
@@ -1,3 +1,8 @@
+# -- DO NOT EDIT THIS FILE --
+# xenserver.conf contains the XenServer specific logging configurations
+# Any remote forwarding should be configured in another file in /etc/rsyslog.d/
+# e.g. /etc/rsyslog.d/remote.conf
+
 # Suppress duplicate messages and report "Last line repeated n times"
 $RepeatedMsgReduction on
 
@@ -86,7 +91,3 @@ local1.*						:omfile:$vmss_log
 # xcp-rrdd-plugins (info and above) to local0
 $outchannel xcp_rrdd_log,/var/log/xcp-rrdd-plugins.log,104857600,/etc/cron.daily/logrotate
 local0.info						:omfile:$xcp_rrdd_log
-
-# ignore default rules
-*.*							~
-
-- 
2.30.2

